torchsnapshot/snapshot.py

```diff
@@ -176,6 +176,7 @@ def take(
         app_state: AppState,
         pg: Optional[dist.ProcessGroup] = None,
         replicated: Optional[List[str]] = None,
+        custom_tensor_prepare_func: Optional[Callable] = None,
```

Comment:
Can you add an explicit type hints for the callable? I think it would be something like `Callable[[str, Tensor, bool], Tensor]`.

Reply:
done!